### PR TITLE
Interaction: alert, prompt, confirm

### PR DESCRIPTION
**Pull request by jonnathan-ls**
# [Interaction: alert, prompt, confirm](https://javascript.info/alert-prompt-confirm)

## 📝 **Goal**

> Article update

## ☑ What was done?

- fixed misspellings of article about alert, prompt, confirm

### CODE REVIEW

**Comment by jonnathan-ls** [comment]
> Error: the article already translated in the Progress Issue #1 ⁉️
> https://github.com/javascript-tutorial/pt.javascript.info/pull/244#issuecomment-1159824552 [javascript-translate-bot](https://github.com/javascript-translate-bot)

There are spelling and syntax errors in our language

File: 1-js/02-first-steps/06-alert-prompt-confirm/article.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Interação: alert, prompt, confirm
 
-Como usuaremos o navegador como nosso ambiente de demonstração, vamos ver algumas funções para interagir com o usuário: `alert`, `prompt` e `confirm`.
+Como usaremos o navegador como nosso ambiente de demonstração, vamos ver algumas funções para interagir com o usuário: `alert`, `prompt` e `confirm`.
 
 ## alert
```
Comment (odsantos) [inline 1-js/02-first-steps/06-alert-prompt-confirm/article.md:6]:
```suggestion
Esta já vimos. Ela mostra uma mensagem e aguarda que o usuário pressione "OK".
```

File: 1-js/02-first-steps/06-alert-prompt-confirm/article.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Por exemplo:
 alert('Olá');
 ```
 
-A mini-janela com a mensagem é chamada de _modal window_. A palavra "modal" significa que o visitante não pode interagir com o resto da página, pressionar outros botões, etc, até que ele tenha lidado com a janela. Nesse caso -- até pressionar "OK".
+A mini-janela com a mensagem é chamada de _modal window_. A palavra "modal" significa que o visitante não pode interagir com o resto da página, pressionar outros botões, etc., até ele ter lidado com a janela. Nesse caso -- até pressionar "OK".
 
 ## prompt
```
Comment (odsantos) [inline 1-js/02-first-steps/06-alert-prompt-confirm/article.md:18]:
```suggestion
O visitante pode digitar algo no campo de entrada do prompt e pressionar OK. Então nós temos esse texto no `result`. Ou ele pode cancelar a entrada pressionando Cancelar ou `key:Esc`, então nós temos `null` como `result`.
```

**Review by odsantos — CHANGES_REQUESTED** [review]
Please, fell free to tell where you disagree. Thanks, Osvaldo.

**Comment by nazarepiedady** [comment]
@jonnathan-ls, could you update this pull request?

File: 1-js/02-first-steps/06-alert-prompt-confirm/article.md (hunk 2 above)
Comment (jonnathan-ls) [reply 1-js/02-first-steps/06-alert-prompt-confirm/article.md:18]: Done in https://github.com/javascript-tutorial/pt.javascript.info/pull/244/commits/e072d452a5d1a24871c5bf9e1fba0ff81ad4a6a5

File: 1-js/02-first-steps/06-alert-prompt-confirm/article.md (hunk 1 above)
Comment (jonnathan-ls) [reply 1-js/02-first-steps/06-alert-prompt-confirm/article.md:6]: Done in https://github.com/javascript-tutorial/pt.javascript.info/pull/244/commits/e072d452a5d1a24871c5bf9e1fba0ff81ad4a6a5

**Comment by jonnathan-ls** [comment]
> ... could you update this pull request? 

/done @nazarepiedady

**Review by nazarepiedady — APPROVED** [review]
@jonnathan-ls, you could defend your choices of translations when you believe that the approach you took can improve the comprehension of the content.

Now, about this pull request, I think it is good to be approved.

I hope to finish the translation of this content even this year and start the editing this year.

**Comment by nazarepiedady** [comment]
@Peruibeloko, we know that you are not a maintainer, but your collaboration has been so pertinent that, to me, it is like you are a maintainer. 

Could you review this pull request?

**Review by Peruibeloko — APPROVED** [review]

**Comment by jonnathan-ls** [comment]
Considering that this PR is about grammatical correction, and with the addition of two approvals @Peruibeloko and @nazarepiedady, plus the adjustments I applied as per @odsantos' review, I am accepting the merge of this PR to apply the update.

**Pull request merged**